src/platform_impl/windows/event_loop.rs

```diff
@@ -330,6 +330,12 @@ impl<T> EventLoopWindowTarget<T> {
   pub fn cursor_position(&self) -> Result<PhysicalPosition<f64>, ExternalError> {
     util::cursor_position().map_err(Into::into)
   }
+
+  #[inline]
+  pub fn set_theme(&self, theme: Option<Theme>) {
+    *self.preferred_theme.lock() = theme;
+    let _ = unsafe { PostMessageW(HWND_BROADCAST, *CHANGE_THEME_MSG_ID, WPARAM(0), LPARAM(0)) };
```

Comment:
Does using `HWND::default()` here not work? because `HWND_BROADCAST` will post to all windows and we want to limit it only to our windows.

Reply:
No, I tried it first and it didn't work

Reply:
Sorry, didn't see we have `owned_windows` here, oops